src/Tokenizer/FCT.php

```diff
@@ -14,78 +14,97 @@
 
 namespace PhpCsFixer\Tokenizer;
 
-/*
- * Forward Compatibility Tokens
+/**
+ * @internal
+ */
+abstract class FCTxPHPAncient
+{
+    public const T_AMPERSAND_FOLLOWED_BY_VAR_OR_VARARG = -1;
+    public const T_AMPERSAND_NOT_FOLLOWED_BY_VAR_OR_VARARG = -2;
+    public const T_ATTRIBUTE = -3;
+    public const T_ENUM = -4;
+    public const T_MATCH = -5;
+    public const T_NULLSAFE_OBJECT_OPERATOR = -6;
+    public const T_PUBLIC_SET = -7;
+    public const T_PROTECTED_SET = -8;
+    public const T_PRIVATE_SET = -9;
```

Comment:
```suggestion
    public const T_PRIVATE_SET = -7;
    public const T_PROTECTED_SET = -8;
    public const T_PUBLIC_SET = -9;
```
Let's sort the here too.